src/ManageCourses.Domain/DatabaseAccess/ManageCoursesDbContext.cs

```diff
@@ -314,6 +314,17 @@ public McOrganisationInstitution GetUserOrganisation(string email, string instCo
             return userOrganisations;
         }
 
+        public IQueryable<McUser> GetMcUsers(string email)
+        {
+            var users = McUsers.FromSql(
+                $"select * from mc_user " +
+                $"where lower(email) = lower(@email)",
+                new NpgsqlParameter("email", email)
+            );
+
+            return users;
+        }
+
```

Comment:
don't we already have this functionality somewhere?

reject

Reply:
ok where!